src/source.rs

```diff
@@ -8,7 +8,26 @@ use std::process;
 use std::thread;
 use ffi as glib_ffi;
 use ffi::{gboolean, gpointer};
-use translate::ToGlib;
+use translate::{FromGlib, ToGlib};
```

Comment:
`use translate::{from_glib, FromGlib, ToGlib};` and use `from_glib` without `Id::`